lib/token_master/model.rb

```diff
@@ -14,7 +14,16 @@ def do_by_token!(klass, key, token, **params)
         model.update!(
           params.merge!(completed_at_col(key) => Time.now)
         )
+        model
+      end
+
+      def force_tokenable!(model, key, **params)
+        check_manageable! model.class, key
+        check_params! key, params
 
+        model.update!(
+          params.merge!(completed_at_col(key) => Time.now)
```

Comment:
More of an fyi, i'd probably default to `.merge` instead of `.merge!` unless there is a reason to merge "in place"